rotkehlchen/assets/utils.py

```diff
@@ -202,9 +211,16 @@ def get_or_create_evm_token(
                 # newly queried data
                 GlobalDBHandler().edit_evm_token(ethereum_token)
             else:
-                userdb.msg_aggregator.add_message(  # inform frontend new token detected
+                # inform frontend new token detected
+                data = {'token_identifier': identifier}
+                if seen is not None:
+                    if seen.tx_hash is not None:
+                        data['seen_tx_hash'] = '0x' + seen.tx_hash.hex()
+                    else:  # description should have been given
+                        data['seen_description'] = seen.description  # type: ignore
```

Comment:
Did we say that we will explain why the type ignore here? I guess that is because data is a `dict[str, str]` instead of `dict[str, Optional[str]]`

Reply:
It's the comment right above.